test/environments/rails71/config/application.rb

```diff
@@ -0,0 +1,20 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative 'boot'
+
+require 'rails/all'
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module RpmTestApp
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults(7.0)
```

Comment:
I believe 7.1 has its own defaults. Could you update this LOC?

Reply:
Thanks! Addressed with [8ea9cef](https://github.com/newrelic/newrelic-ruby-agent/pull/2740/commits/8ea9cef0132f714ee5ef7e146ed6090800edd2b0)